package.json

```diff
@@ -2,8 +2,8 @@
   "name": "aqts-capture-error-handler",
   "description": "Deployment via serverless",
   "dependencies": {
-    "serverless": "^1.64.0",
+    "serverless": "^1.65.0",
     "serverless-python-requirements": "^5.1.0",
-    "serverless-plugin-git-variables": "^3.4.0"
+    "serverless-plugin-git-variables": "^4.0.0"
```

Comment:
I think I may have missed this before - but looks like we've moved the install of serverless from an explicit call in the jenkinsfile to just defining it in the package.json.  It then gets installed with all the other dependencies with `npm install` right?

Reply:
Yep. I would have preferred to do it this way from the beginning but it never worked quite right until now.